gateway/test/dapi_alpha_test.go

```diff
@@ -11,3 +14,757 @@ func (s *GatewayOpsTestSuite) TestDapiAlphaEnabled() {
 	})
 	requireRestSuccess(s.T(), resp)
 }
+
+func (s *GatewayOpsTestSuite) insertBinaryDocument() string {
```

Comment:
This should probably be using one of the existing test functions that exist for pushing document contents to the cluster for tests.  Alternatively if there is some specific reason for having this use the DAPI mechanism, it should likely use a normal POST with Content-Type of binary to reflect the actual name of the method (insert as opposed to increment).